content/200-concepts/100-components/01-prisma-schema/06-relations/420-relation-mode.mdx

```diff
@@ -0,0 +1,176 @@
+---
+title: 'Relation mode'
+metaTitle: 'Relation mode'
+metaDescription: 'Relation mode'
+tocDepth: 2
+preview: true
+---
+
+<TopBlock>
+
+In relational databases, relations between data are usually implemented with foreign keys. A relation consists of a target row in a table, and a foreign key field that points to that row.
+
+Foreign keys:
+
+- ensure referential integrity (Can this foreign key field hold this value?)
+- allow you to set referential actions that define how changes to these fields are handled (What happens if the ID of this field is changed?).
+
+In Prisma, relations are defined using the [`@relation`](/reference/api-reference/prisma-schema-reference#relation) attribute, which then under the hood is translated to foreign keys and the SQL to define referential actions (which are defined with the `onUpdate` and `onUpdate` arguments of the attribute).
+
+!!explain example
+
+<!--
+Prisma allows you to define relations between records using the [`@relation`](/reference/api-reference/prisma-schema-reference#relation) attribute. The `onUpdate` and `onDelete` arguments of this attribute can be used to define which referential action take place on updating or deleting the related record.
+-->
+
+<!-- TODO Tab that shows SQL -->
+
+```prisma file=schema.prisma highlight=4;normal
+model Post {
+  id       Int    @id @default(autoincrement())
+  title    String
+  author   User   @relation(fields: [authorId], references: [id], onDelete: Cascade)
+  authorId Int
+}
+
+model User {
+  id    Int    @id @default(autoincrement())
+  posts Post[]
+}
+```
+
+Some databases, such as MongoDB or [PlanetScale](/guides/database/using-prisma-with-planetscale#differences-to-consider)),
+do not support foreign keys. For this situation, Prisma offers an alternative _relation mode_ where most properties of relations in relational databases are emulated.
```

Comment:
```suggestion
Some databases, such as MongoDB or [PlanetScale](/guides/database/using-prisma-with-planetscale#differences-to-consider)), do not support foreign keys. For this situation, Prisma offers an alternative _relation mode_ where most properties of relations in relational databases are emulated.
```
technically not "of relations" but "of foreign keys". 
but maybe then it would need to be "functionalities" (which are constraints and referential actions) instead of "properties".